### PR TITLE
Schema update 1.2.0

### PR DESCRIPTION
**Pull request by this-username-is-taken**

### CODE REVIEW

File: subgraphs/_reference_/src/common/constants.ts [hunk 1]
```diff
@@ -23,6 +26,27 @@ export namespace Network {
   export const XDAI = "XDAI";
 }
 
+// The network names corresponding to the ones in `dataSource.network()`
+// They should mainly be used for the ease of comparison.
+// See below for a complete list:
+// https://thegraph.com/docs/en/hosted-service/what-is-hosted-service/#supported-networks-on-the-hosted-service
+export namespace SubgraphNetwork {
```
Comment (steegecs) [inline subgraphs/_reference_/src/common/constants.ts:33]: Missing "harmony" and "cronos"

Comment (this-username-is-taken) [reply subgraphs/_reference_/src/common/constants.ts:33]: yeah, I don't think these two are supported by The Graph yet

Comment (this-username-is-taken) [reply subgraphs/_reference_/src/common/constants.ts:33]: see https://thegraph.com/docs/en/hosted-service/what-is-hosted-service/#supported-networks-on-the-hosted-service

Comment (this-username-is-taken) [reply subgraphs/_reference_/src/common/constants.ts:33]: I don't mind removing them from the list above

**Pull request closed without merging**